foia_hub/scripts/load_agency_contacts.py

```diff
@@ -38,9 +38,16 @@ def check_urls(agency_url, row, field):
 def clean_phone(number_str):
     """Cut down the phone number string as much as possible. If multiple
     numbers are present, take the first only"""
+
     number_str = number_str or ''
-    number_str = number_str.replace('(', '').replace(')', '')
-    number_str = number_str.replace('ext. ', 'x').replace('ext ', 'x')
+
+    if '+' in number_str:
+        return number_str
+
+    number_str = number_str.replace('(', '')
+    number_str = re.sub('[)][- ]?', '-', number_str)
+    number_str = number_str.replace(' ', '')
+    number_str = re.sub('[, ]*ext[. ]*', ' x', number_str)
     number_str = number_str.split(',')[0].strip()
```

Comment:
Is there any reason why we only pull the first number into the model? Should we should consider a different model relationship?